DATAUP-246, css to scss: remove unneeded dir

ialarmedalien opened the pull request:
# Description of PR purpose/changes

Removing html_mockups folder, which has not been edited in five years.

# Jira Ticket / Issue #
https://kbase-jira.atlassian.net/browse/DATAUP-246

briehl reviewed (approved):
I'm for removing these from the repo, but can we keep / stash the kbase_styles.html file in Google Docs or somewhere? That's currently how things are styled, and it would be nice to have to point to.

ialarmedalien commented:
I've moved the style guide into the `kbase-extension/static/kbase/template/`, so you can view it if you manually enter the template address into your browser (i.e. `http://localhost:8888/narrative/static/kbase/templates/kbase_styles.html`). I've also updated it to use the current stylesheet.
